extensions/Worker.Extensions.CosmosDB/CosmosDBInputAttribute.cs

```diff
@@ -25,27 +25,28 @@ public CosmosDBInputAttribute(string databaseName, string collectionName)
         public string DatabaseName { get; private set; }
 
         /// <summary>
-        /// The name of the collection to which the parameter applies.
+        /// The name of the container to which the parameter applies.
         /// May include binding parameters.
         /// </summary>
-        public string CollectionName { get; private set; }
+        public string ContainerName { get; private set; }
 
         /// <summary>
-        /// Optional. A string value indicating the app setting to use as the CosmosDB connection string, if different
-        /// than the one specified in the <see cref="CosmosDBOptions"/>.
+        /// Optional.
+        /// Gets or sets the connection string for the service containing 
```

Comment:
Is this supposed to the the actual connection string or the reference to the setting? This should be a reference (as we wouldn't persist the actual connection string in metadata), which would mean that we'd need to make sure the documentation here is clear, but it would be good to confirm.

@ealsur might be able to confirm

Reply:
I copied over the description from here: https://www.fuget.org/packages/Microsoft.Azure.WebJobs.Extensions.CosmosDB/4.0.0-preview2/lib/netstandard2.0/Microsoft.Azure.WebJobs.Extensions.CosmosDB.dll/Microsoft.Azure.WebJobs/CosmosDBAttribute 

No clear answer from this doc.

Reply:
I believe this is the reference, at least according to this documentation:

```
The name of the app setting containing your Azure Cosmos DB connection string.

In version 4.x of the extension this property is called Connection. The value is the name of an app setting that either contains the connection string or contains a configuration section or prefix which defines the connection. See Connections.
```

https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-cosmosdb-v2-output?tabs=csharp#configuration

Reply:
It should be the name of the setting that contains the connection information.